### PR TITLE
[FA-987] add alert key

### PR DESCRIPTION
**Pull request by Olson3R**
# [FA-987] add alert key
## What?
Added the optional key parameter when creating an alert.
## Why?
Would like to use keys when creating alerts.
## QA Strategy
- [x] Merged latest master
- [x] Updated version number
- [x] Can create alerts with a key
- [x] Can create alerts without a key
## Breaking Changes
- None

### CODE REVIEW

**Comment by cliff-dellamano** [comment]
Looks good to me, but I'm not sure I fully grasp what this `key` parameter is accomplishing. I read through the thread on Slack and it appears it's going to be used to prevent multiple alerts for events that only happen once per season, but it's not clear to me how this solves that problem if an alert can be created with a `null` value for `key`. Or is the idea to use it within logic on the FieldAgent backend?

File: sentera/api.py [hunk 1]
```diff
@@ -111,16 +111,18 @@ def create_alert(field_sentera_id, name, message, token):
     :param token: an authorization token needed to post the alert to the specified field (string)
     :return: result of the request.post
```
Comment (elliott-imhoff) [inline sentera/api.py]: Maybe add `key` to the param list

File: sentera/api.py [hunk 2]
```diff
@@ -111,16 +111,18 @@ def create_alert(field_sentera_id, name, message, token):
     :param token: an authorization token needed to post the alert to the specified field (string)
```
Comment (Olson3R) [inline sentera/api.py]:
```suggestion
    :param token: an authorization token needed to post the alert to the specified field (string)
    :param key: (optional) A client-defined key to help identify the alert.
```

File: sentera/api.py (hunk 1 above)
Comment (Olson3R) [reply sentera/api.py]: Added.

**Comment by jdalt** [comment]
- [ ] Passes QA via commandline magic.

**Pull request merged**